java-tic-tac-toe.iml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Usually we don't check in the generated intellij files. You should add *.iml to the .gitignore